### PR TITLE
Add lograge to format production logs nicely

### PR DESCRIPTION
**Pull request by cbeer**

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -25,12 +25,21 @@ class ApplicationController < ActionController::Base
...
 
     def multitenant?
       Settings.multitenant
     end
+
+    def current_account
+      @current_account ||= Account.from_request(request)
+    end
+
+    def append_info_to_payload(payload)
```
Comment (jcoyne) [inline app/controllers/application_controller.rb]: Add a comment here explaining why this method is used.

Comment (cbeer) [inline app/controllers/application_controller.rb]: 👌

**Comment by jcoyne** [comment]
Needs a rebase.

**Comment by jcoyne** [comment]
rebase again

**Pull request merged**